### PR TITLE
Matt front end explore

### PR DESCRIPTION
**Pull request by mnolan34**
Simple Adding Dropdown selection and Search Button to Front End. Next Step is to add backend/state functionality to both.

### CODE REVIEW

File: src/components/tuiter/index.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ function Tuiter () {
               <Route path="/tuiter/:uid" element={<Home/>}/>
               <Route path="/home" element={<Home/>}/>
               <Route path="/home/:uid" element={<Home/>}/>
+              <Route path="/explore" element={<Explore/>}/>
               <Route path="/explore/*" element={<Explore/>}/>
```
Comment (mcartervt) [inline src/components/tuiter/index.js:37]: Looks like this is here twice now?

**Review by mcartervt — CHANGES_REQUESTED** [review]
Just checkout my comment from components/tuiter/index.js

Otherwise looks great!

File: src/components/tuiter/index.js (hunk 1 above)
Comment (mnolan34) [reply src/components/tuiter/index.js:37]: This is on purpose, as /explore will be the basic "explore" page, while /explore/* will allow us to alter the explore url to have explore/BBQLife to fill in BBQLife tuits and have the value in the search bar.

**Review by mcartervt — APPROVED** [review]

**Comment by kachalmers** [comment]
Warning exists for search button, but it can be fixed later after the button functionality is implemented. Approved for pull request.

**Pull request merged**